src/components/WebexRemoteMedia/WebexRemoteMedia.js

```diff
@@ -17,15 +16,40 @@ import './WebexRemoteMedia.scss';
  * NOTE: waiting for the UX for a design on what to display if there is no remote video
  */
 export default function WebexRemoteMedia({className, meetingID}) {
-  const {remoteAudio, remoteVideo, error} = useMeeting(meetingID);
+  const {remoteAudio, remoteVideo, remoteShare, error} = useMeeting(meetingID);
+  const [, shareControl] = useMeetingControl('share-screen');
   const audioRef = useStream(remoteAudio);
   const videoRef = useStream(remoteVideo);
-  const hasMedia = !!(remoteAudio || remoteVideo);
+  const shareRef = useStream(remoteShare);
+  const hasMedia = !!(remoteAudio || remoteVideo || remoteShare);
   const mainClasses = {
     [`${WEBEX_COMPONENTS_CLASS_PREFIX}-remote-media`]: true,
     [className]: !!className,
   };
 
+  const isLocalSharingInactive = shareControl.state === 'inactive';
+
+  // Switch the visibility of the remoteShare video by checking its readyState
+  // Note: Sharing detection can be removed from component, once the sdk provides
+  //       the Start/Stop events about the remote sharing.
+  //
+  const detectRemoteSharing = () => {
+    if (!shareRef.current) {
+      return false;
+    }
+
+    return shareRef.current.readyState > 1;
+  };
+
+  const remoteShareClasses = {
```

Comment:
Are these necessary? If they are, this should probably be unified at the component level in [`mainClasses`](https://github.com/webex/components/blob/master/src/components/WebexRemoteMedia/WebexRemoteMedia.js#L24)

Reply:
It is used to stylized one single video element inside the `<WebexRemoteMedia>`, not the whole component. However, the video element itself is not big enough to be a standalone component.

Reply:
The `remoteShareClasses` will be removed. Instead, we choose a better way to do split screen.